templates/starter/bicep/README.md

```diff
@@ -7,3 +7,65 @@ The following assets have been provided:
 - Infrastructure-as-code (IaC) files under the `infra` folder that demonstrate how to provision resources and setup resource tagging for `azd`.
 - A [dev container](https://containers.dev) configuration file under the `.devcontainer` directory that installs infrastructure tooling by default. This can be readily used to create cloud-hosted developer environments such as [GitHub Codespaces](https://aka.ms/codespaces).
 - Continuous deployment workflows for CI providers such as GitHub Actions under the `.github` directory, and Azure Pipelines under the `.azdo` directory that work for most use-cases.
+
+## Next Steps
+
```

Comment:
Can we link to the additional details section here?

Reply:
Let me know if the current changes still need work